src/main/java/htsjdk/samtools/util/BlockGunzipper.java

```diff
@@ -43,10 +44,32 @@
  * @author alecw@broadinstitute.org
  */
 public class BlockGunzipper {
-    private final Inflater inflater = new Inflater(true); // GZIP mode
+    private static InflaterFactory defaultInflaterFactory = new InflaterFactory();
```

Comment:
In addition to this default inflater set via a static method, could we also have a non-static method on `SamReaderFactory` called `setInflaterFactory()` that can override the statically-set default? This would mirror what was done for `DeflaterFactory`.